django/contrib/admin/static/admin/css/forms.css

```diff
@@ -83,7 +83,7 @@ form ul.inline li {
     height: 26px;
 }
 
-.aligned label + p, .aligned label + div.help {
+.aligned label + p, .aligned label + div.readonly {
```

Comment:
Should this rule be duplicated with div.readonly rather than removing the div.help version?